exponent_server_sdk/__init__.py

```diff
@@ -118,6 +121,9 @@ def get_payload(self):
             payload['priority'] = self.priority
         if self.badge is not None:
             payload['badge'] = self.badge
+        if self.channel_id is not None:
+            payload['channel_id'] = self.channel_id
```

Comment:
The field in the JSON-encoded payload sent to the server should be channelId instead of channel_id